client/state/data-layer/wpcom/read/streams/index.js

```diff
@@ -243,16 +257,33 @@ export function handlePage( action, data ) {
 		// and offsets must be used
 		const { after } = date_range;
 		pageHandle = { before: after };
+	} else if ( next_page_handle ) {
+		pageHandle = { page_handle: next_page_handle };
+	}
```

Comment:
What did this fix for us? Looking at this, it seems like `next_page` handles things the same way above.. i wonder why there was a separate `next_page_handle` needed by whatever might be using it?